packages/dev/core/src/Shaders/ShadersInclude/helperFunctions.fx

```diff
@@ -45,34 +45,94 @@ mat3 inverseMat3(mat3 inMatrix) {
               b21, (-a21 * a00 + a01 * a20), (a11 * a00 - a01 * a10)) / det;
 }
 
+#if USEEXACTSRGBCONVERSIONS
+vec3 toLinearSpaceExact(vec3 color)
+{
+    vec3 nearZeroSection = 0.0773993808 * color;
+    vec3 remainingSection = pow(0.947867299 * (color + vec3(0.055)), vec3(2.4));
+    #if defined(WEBGL2) || defined(WEBGPU)
+        return mix(remainingSection, nearZeroSection, lessThanEqual(color, vec3(0.01)));
+    #else

```

Comment:
Note the difference in these cases is due to the fact glsl 1.0 doesn't support a bvec as the interpolation factor (but glsl 3.0 does). I'm assuming that means WebGPU can handle this too.

Reply:
Yes, this will be correctly converted to WGSL by the GLSL=>WGSL converter.